pkg/controller/deployment/deployment_controller.go

```diff
@@ -464,14 +466,14 @@ func (dc *DeploymentController) classifyReplicaSets(deployment *extensions.Deplo
 				// If the RS no longer exists, don't even log the error.
 				if !errors.IsNotFound(err) {
 					utilruntime.HandleError(err)
```

Comment:
We can probably also remove this HandleError since the aggregate error will be logged or passed to HandleError if the deployment is about to be dropped out of the queue (see `func (dc *DeploymentController) handleErr(err error, key interface{})`)